Modix.Bot/Behaviors/ModerationLoggingBehavior.cs

```diff
@@ -91,6 +99,15 @@ internal protected IModerationService ModerationService
             => _lazyModerationService.Value;
         private readonly Lazy<IModerationService> _lazyModerationService;
 
+        internal static protected ModixConfig Config { get; private set; }
```

Comment:
Hmm. Not sure about using `static` for a non-readonly/injected property in a scoped class like this.

Reply:
I think I had `static` there because it was having trouble seeing it from within the dictionary. I've removed static and now it seems happy with it. I think R# expiring may be causing some weird intellisense issues.

Reply:
Oh, no, actually it was R# suggesting I make it static. 
![image](https://user-images.githubusercontent.com/7003081/57102065-3346c800-6ce8-11e9-8deb-c4807a57423c.png)